digdag-standards/src/main/java/io/digdag/standards/operator/ShOperatorFactory.java

```diff
@@ -128,28 +128,41 @@ private void runCode(final Config state)
             }
         }
 
+        private List<String> getShell(final Config params, final Boolean winOS){
+            // Until digdag ver. 0.9, it was necessary to write ["powershell.exe","-"] to specify shell, 
+            // but since ver. 0.10, jobs are not executed unless ["powershell.exe"] is written.
+            // To resolve this incompatibility, change the shell specification ["powershell.exe","-"]
+            // to ["powershell.exe"].            
+            List<String> temp_shell;
+            if (params.has("shell")) {
+                temp_shell = params.getListOrEmpty("shell", String.class);
+                if (temp_shell.get(0).toLowerCase().contains("powershell") && temp_shell.size() == 2){
```

Comment:
`contains("powershell")` will match to strings like 'powershell-batch.bat`. Is it intentional ?